scripts/fly_sync_and_login.sh

```diff
@@ -2,22 +2,47 @@
 set -euo pipefail
 
 # Required env vars
-# CONCOURSE_URL
-# CONCOURSE_ATC_USER
-# CONCOURSE_ATC_PASSWORD
-# FLY_CMD
-# FLY_TARGET
+# shellcheck disable=SC2086
+: $CONCOURSE_URL \
```

Comment:
Nice trick!

Reply:
Necessity is the mother of invention ... mostly driven by my desire not to have 

```
# shellcheck disable=SC2086
: $VAR1
# shellcheck disable=SC2086
: $VAR2
# shellcheck disable=SC2086
: $VAR3
```

:-)